.github/workflows/ci.yml

```
@@ -401,7 +403,7 @@ jobs:
 
   release_assets:
```

Minor nit I didn't notice before; this should be `kabab-case` not `snake_case`.

```suggestion
  release-assets:
```

---

Fixed a bit more thoroughly in: https://github.com/deskflow/deskflow/pull/7703